### ISSUE TITLE
Typo when overriding endpoint?

### ISSUE THREAD

**Issue by euan-forrester** [multiplegeorges/vue-cli-plugin-s3-deploy#121]
I noticed this line: 

https://github.com/multiplegeorges/vue-cli-plugin-s3-deploy/blob/master/src/deployer.js#L24

Should it instead be `config.options.endpoint`? There's no mention of `overrideEndpoint` anywhere in the readme

**Comment by nicekiwi** [comment]
Hmm not sure about this one, as there are un-documented features floating around in the @next branch. `endpoint` would make sense. But I'm ignoring the current releases as much as possible as the project structure changes quite a bit between current and now `rc5`.

**Comment by euan-forrester** [comment]
Fair enough :). Doesn't seem to have tripped anyone up yet. Looking forward to seeing what's coming up!
